src/pyodbc.pyi

```diff
@@ -451,5 +451,5 @@ def connect(connstring: Optional[str] = None,
             ansi: bool = False,
             readonly: bool = False,
             timeout: int = 0,
-            attrs_before: dict = {},
-            **kwargs) -> Connection: ...
+            attrs_before: Dict[Any, Any] = {},
```

Comment:
Could we change this to `Dict[int, Any]`?  The dict keys on `attrs_before` must be integers.  The dict values can be a variety of types so best leave that as `Any` for now.

Reply:
Done.